roles/openshift_node/meta/main.yml

```diff
@@ -13,4 +13,5 @@ galaxy_info:
   - cloud
 dependencies:
 - { role: openshift_common }
+- { role: openshift_repos }
 - { role: docker }
```

Comment:
Hmm, I forgot this dependency was in here...  I'm not sure we want a dependency on the docker role unless it is a containerized host, but that would have to be playbook driven and not metadata driven, since I don't think you can add conditionals on roles in the metadata.